gym/envs/mujoco/mujoco_env.py

```diff
@@ -101,12 +101,22 @@ def do_simulation(self, ctrl, n_frames):
         for _ in range(n_frames):
             self.sim.step()
 
-    def render(self, mode='human', width=DEFAULT_SIZE, height=DEFAULT_SIZE):
+    def render(self,
+               mode='human',
+               width=DEFAULT_SIZE,
+               height=DEFAULT_SIZE,
+               camera_id=None,
+               camera_name=None):
         if mode == 'rgb_array':
-            camera_id = None 
-            camera_name = 'track'
-            if self.rgb_rendering_tracking and camera_name in self.model.camera_names:
-                camera_id = self.model.camera_name2id(camera_name)
+            if camera_id is None or camera_name is None:
+                raise ValueError("Both `camera_id` and `camera_name` cannot be"
+                                 " specified at the same time.")
+
+            no_camera_specified = camera_name is None and camera_id is None
```

Comment:
This should maintain the backwards compatibility. I don't know if it's needed though. Could we just get rid of the `rgb_rendering_tracking`?

Reply:
Get rid if `rgb_rendering_tracking` - i.e. use `camera_name='track'` instead? Yes, we can do that (probably will merit a new version release, but otherwise I am not opposed

Reply:
Done. I added a deprecation warning for it for now.